backend/tags/filters/taggable.py

```diff
@@ -1,13 +1,11 @@
 import django_filters
-from django.db.models import Count, Q
+from django.db.models import Q
 from django_filters import rest_framework as filters
 
-from tags.models import Tag
-
 """
-Taggable filter allows to filter taggable objects with an intersection of tags. If you're working with Media for 
-instance, /associations/medias/?tag_id__in=1,2 will return all medias that have a tag with id 1 or 2.
-If you want to query all medias with tag 1 AND 2, use /associations/medias/?tag__are=1,2
+Taggable filter allows to filter taggable objects with an intersection of tags. If you're working with Media 
+for instance, `/associations/medias/?tag__are=1,2` will query all medias with tag 1 AND 2.
```

Comment:
Is it `tags` ?
Also, I find it a bit confusing that it's tag__in (no plural) and tags__are (plural)